tests/tests/worm-xattr-unit.sh

```diff
@@ -0,0 +1,49 @@
+t_require_commands touch rm setfattr
+
+echo "== test creation"
+touch "$T_D0/file-1"
+SECS=$(date '+%s')
+NSECS=$(date '+%N')
+DELAY=10
+EXP=$(( $SECS + $DELAY ))
+setfattr -n scoutfs.worm.v1_expiration -v $EXP.$NSECS "$T_D0/file-1"
+
+echo "== test value is set correctly"
+getfattr --absolute-names -d -m - "$T_D0/file-1" | grep $EXP.$NSECS &> /dev/null
+if [ $? == 0 ]; then
+    echo "== getfattr matches setfattr"
```

Comment:
Adding a test heading as matching output is a bit confusing.. I'd do something like || echo "getfattr doesn't match".  Arguably better still would be to use diff and bash tricks so we only get output when they don't match and it'll show us how they differ.   Something like diff -u <(echo "$EXP.$NSECS") <(getfattr --absolute-name --value-only -n scoutfs.worm.v1_expiration $T_D....).